source/ui/BioXAS/BioXASControlEditor.cpp

```diff
@@ -365,6 +379,15 @@ void BioXASControlEditor::onCalibrateActionTriggered()
 	}
 }
 
+void BioXASControlEditor::onPropertiesActionTriggered()
+{
+	if (control_) {
+		AMControlView *view = new AMControlView(control_);
+		view->setWindowTitle(QString("%1 properties").arg(title()));
+		view->show();
```

Comment:
You're right that BioXASControlEditor doesn't keep track. Would a better solution be to listen for a closed() type signal and explicitly delete this widget then?